dist/common/scripts/scylla_coredump_setup

```diff
@@ -99,6 +99,7 @@ WantedBy=multi-user.target
         try:
             run('coredumpctl --no-pager --no-legend info {}'.format(pid))
             print('\nsystemd-coredump is working finely.')
+            run('rm -f /var/lib/systemd/coredump/core.bash.*')
```

Comment:
Who said that all core.bash* are meaningless, I think I was misunderstood, I think we should delete **only**
the cores that were created in this script or we might delete unrelated legitimate cores.